lib/tasks/publishing_api.rake

```diff
@@ -234,6 +234,18 @@ namespace :publishing_api do
       puts "Finished queuing items for Publishing API"
     end
 
+    desc "Republish all documents with draft editions"
+    task all_drafts: :environment do
+      editions = Edition.draft.includes(:document)
```

Comment:
```suggestion
      editions = Edition.in_pre_publication_state.includes(:document)
```
There are a few document states (e.g. `submitted`) which are not draft but where a `auth_bypass_id` will be present.  The [`in_pre_publication_state` scope](https://github.com/alphagov/whitehall/blob/9eb0010f6472346b2a99bcddb70c63f874ed981d/app/models/edition.rb#L83) should cover those.